routing.rst

```diff
@@ -794,6 +794,52 @@ parameter:
     To give a ``null`` default value to any parameter, add nothing after the
     ``?`` character (e.g. ``/blog/{page?}``).
 
+Priority Parameter
+~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.1
+
+    The ``priority`` parameter was introduced in Symfony 5.1
+
+When defining a greedy pattern that matches many routes, this may be
+at the beginning of your routing collection and prevents any route defined after to be matched.
+A ``priority`` optional parameter is available in order to let you choose the order of your routes,
+and it is only available when using annotations to define routes.
+
+.. code-block:: php-annotations
+
+    // src/Controller/BlogController.php
+    namespace App\Controller;
+
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+    use Symfony\Component\Routing\Annotation\Route;
+
+    class BlogController extends AbstractController
+    {
+        // ...
+
+        /**
```

Comment:
I would add a comment in the doc block before the route definition, stating some thing like:
```
/*
 * Using an extremely low priority to ensure that the matcher will test it as very last
 *
 * @Route(...)
```